models/product-selection/src/transformers.ts

```diff
@@ -8,10 +8,15 @@ const transformers = {
   rest: Transformer<TProductSelection, TProductSelection>('rest', {
     buildFields: ['createdBy', 'lastModifiedBy', 'name'],
   }),
-  // graphql scaffolding only
+
   graphql: Transformer<TProductSelection, TProductSelectionGraphql>('graphql', {
-    buildFields: ['createdBy', 'lastModifiedBy', 'name'],
-    addFields: () => ({
+    buildFields: ['createdBy', 'lastModifiedBy'],
+    removeFields: ['name'],
+    addFields: ({ fields }) => ({
+      nameAllLocales: buildField(fields.name, 'graphql', {
+        fieldToBuild: 'name',
+      }),
+      productRefs: null,
```

Comment:
this would return an array of products, but I'm not sure of the shape of what's returned and tbh it seems out of scope to add this to the builder for this PR.  It should probably be added separately (ideally when needed by the team that needs it)